### DISCUSSION TITLE
How does it work in browser?

### DISCUSSION THREAD

**Discussion by dragonfly-ai** [stdlib-js/stdlib#1130] [General]
From the README.md: "... written in JavaScript (and C) for execution in browsers and in Node.js ..."

Can I safely assume that the parts written in C only run from Node.js, or can the Browser somehow access C libraries?  If so, how!?

**Comment by kgryte** [comment]
@dragonfly-ai Thanks for your question. You are correct; the parts written in C only run in Node.js. All C APIs have a JavaScript fallback which allows usage in browsers.
